app/models/project.rb

```diff
@@ -92,6 +106,27 @@ class Project < ActiveRecord::Base
   ranks :launched_row_order
   ranks :beta_row_order
 
+  def self.scope_for(action, user, opts={})
+    experiment_name = "eager_load_projects"
+    scope = super
+    CodeExperiment.run "#{experiment_name}" do |e|
+      e.run_if { Panoptes.flipper[experiment_name].enabled? }
+      e.context user: user
+      e.use { scope }
+      e.try do
+        if opts[:skip_eager_load]
```

Comment:
I can see the clarity of using a keyword arg but moving to a keyword arg means i have to modify the method sigs of a lot of classes that don't use the eager load code (yet). The options hash currently allows to modify runtime behaviour without touching method sigs.

If we refactor all the controlled resources to have eager loading then yeah we should refactor the method sigs as well. 

I'll update to use the positive voice.